src/Sarif/Errors.cs

```diff
@@ -128,6 +129,26 @@ public static void LogAllRulesExplicitlyDisabled(IAnalysisContext context)
 
             context.RuntimeErrors |= RuntimeConditions.NoRulesLoaded;
         }
+        public static void LogNoPluginsConfigured(IAnalysisContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            // No analysis plugins were configured, therefore no rules loaded.
+            context.Logger.LogConfigurationNotification(
+                    Errors.CreateNotification(
+                        context.TargetUri,
+                        ERR997_NoPluginsConfigured,
+                        ruleId: null,
+                        FailureLevel.Error,
+                        exception: null,
+                        persistExceptionStack: false,
+                        messageFormat: null));
+
+            context.RuntimeErrors |= RuntimeConditions.NoRulesLoaded;
```

Comment:
>NoRulesLoaded [](http://example.com/codeflow?start=55&length=13)

should we add a dedicated RuntimConditions value for no plugin error?

Reply:
dedicated RuntimCondition may add some value for person who is troubleshooting.